lib/features/homeworks/lesson_19/homework_cubit/homework_cubit_screen.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_lab/features/homeworks/lesson_19/homework_cubit/cubit/counter_cubit.dart';
+import 'package:flutter_lab/features/homeworks/lesson_19/number_input_field.dart';
+
+class HomeworkCubitScreen extends StatefulWidget {
+  const HomeworkCubitScreen({super.key});
+
+  @override
+  State<HomeworkCubitScreen> createState() => _HomeworkCubitScreenState();
+}
+
+class _HomeworkCubitScreenState extends State<HomeworkCubitScreen> {
+  final TextEditingController _controller = TextEditingController(text: '0');
```

Comment:
```suggestion
  final  _controller = TextEditingController(text: '0');
```
Type inference works here, so there’s no need to explicitly declare the type.